lib/fluent/plugin/in_systemd.rb

```diff
@@ -45,22 +67,22 @@ def shutdown
       private
 
       def init_journal
-        @journal = Systemd::Journal.new(path: @path)
+        @journal = ::Systemd::Journal.new(path: @path)
```

Comment:
Originally, I had namespaced the entry mutator like `Fluent::Plugin::Systemd`. Given that the input plugin is declared in `Fluent::Plugin`, `Systemd` referenced the module relative to the local namespace. I have since changed the mutator class to this fully qualified name: `Fluent::Plugin::SystemdEntryMutator`